### DISCUSSION TITLE
Are multiple (native) windows possible?

### DISCUSSION THREAD

**Discussion by pbouffard** [Gnimuc/CImGui.jl#40] [Q&A] [answered]
In my case it would be on MacOS but it would be good to know the answer for all the supported platforms too.

**Comment by Gnimuc** [comment, marked as answer]
What do you mean by "multiple native windows"? Do you mean "multiple GLFW windows" or "multiple imgui contexts" or both?

In the demo folder, there is an example of multiple GLFW windows +multiple imgui contexts, but the recommended way seems to be using a single imgui context with [multi-viewport](https://github.com/ocornut/imgui/issues/1542). However, the multi-viewport feature is not supported in CImGui.jl for now, see #34.
